apps/frontend/src/views/Login.vue

```diff
@@ -91,5 +95,25 @@ export default class Login extends Vue {
   get ldapenabled() {
     return ServerModule.ldap;
   }
+
+  get logoffSnackbar() {
+    const queryString = window.location.search;
+    const urlParams = new URLSearchParams(queryString);
+    if (
+      urlParams.get('logoff')?.toLowerCase() === 'true' &&
+      ServerModule.token === ''
+    ) {
+      return true;
+    } else if (
+      urlParams.get('logoff')?.toLowerCase() === 'true' &&
+      ServerModule.token !== ''
+    ) {
+      this.logoffMessage =
+        'An error occoured while logging you out and your token has not been discarded. Please clear your browsing data.';
```

Comment:
occoured is misspelled - Suggest you use the following message:
An error occurred during the logout process, your token has been discarded. Please clear the browser data (history)